tempesta_fw/sock_clnt.c

```diff
@@ -451,6 +529,13 @@ TfwCfgMod tfw_sock_clnt_cfg_mod  = {
 			.allow_repeat = true,
 			.cleanup = tfw_sock_clnt_cfg_cleanup_listen
 		},
+		{
+			"keepalive_timeout",
+			"75",
+			tfw_sock_clnt_cfg_handle_keepalive,
+			.allow_repeat = true,
```

Comment:
Right now the keep-alive timeout is the one and only for all listening ports. So this option should not be allowed to berepeated in the configuration file.